### ISSUE TITLE
Git commands permission denied makes build fail

### ISSUE THREAD

**Issue by nicooprat** [jacobmarshall/webpack-bugsnag-plugin#9]
So, I'm using Netlify to deploy my app. They have permissions that don't allow git commands. I tried to set some options manually to avoid having to get them with Git (I won't be able to set `revision` though):

```js
new BugsnagDeployPlugin({
  apiKey: '...',
  releaseStage: 'production',
  repository: '...',
  branch: 'prod'
}),
```

But it's still trying to run some Git commands that fail from the [`getProjectDetails`](https://github.com/jmshal/webpack-bugsnag-plugin/blob/master/src/helpers/CommonBugsnagPlugin.js#L35) function. Is there any way to make the [functions in git helper](https://github.com/jmshal/webpack-bugsnag-plugin/blob/master/src/helpers/git.js#L4) more resistant to errors? I tried a few things without success...

Thanks.

**Comment by jmshal** [comment]
Hi @nicooprat,

Thanks for the issue - I really would love to get this working for you! I have published a new version (v0.1.4), which I hope will resolve the volatility of those helper functions with permission issues.

Please let me know if there is anything else. After Christmas break I will revisit this and see whether I can utilise something like nodegit to get around those permission issues. But for now this is the best I can do.

Thanks again :)

**Comment by nicooprat** [comment]
Hi @jmshal 

Thanks for your quick answer. Looks like everything is working fine! 👍
